scripts/test_collection/tt.sh

```diff
@@ -0,0 +1,234 @@
+#!/bin/bash
+
+# ======================================================
```

Comment:
I wasn't sure of the purpose of this script vs the other one? It's similar but not quite the same.